### DISCUSSION TITLE
22일까지 일정 정하기

### DISCUSSION THREAD

**Discussion by YuHyun-P** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#162] [Ideas]
API 연결, 모든 페이지 merge 후 테스트, 배포 등 많은 일이 남았는데 각 할 일들의 기한을 정하면 좋을 것 같아 안건으로 올립니다!

**Comment by Hyevvy** [comment]
넵 !! 좋아용 🥰

**Comment by YuHyun-P** [comment]
✅ 20일 자정까지 모든 페이지 develop 브랜치로 merge!
